src/entries/popup/components/IconAndCopyList.tsx/IconAndCopyList.tsx

```diff
@@ -39,6 +46,16 @@ export function IconAndCopyList({
             </Box>
             <Text size="14pt" weight="medium" color="label">
               {copy}
+              {link && (
+                <TextLink
+                  color="blue"
+                  onClick={() =>
+                    goToNewTab({ url: 'https://learn.rainbow.me/' })
```

Comment:
shouldn't you be using `href` prop for this link? feels like this is adding a hardcoded link to all `IconAndCopyList` components that may not even be related to that link